### ISSUE TITLE
OL wssecurity CXF FAT with EE8 and Java11 failed with NPE in FFDC

### ISSUE THREAD

**Issue by meiaus** [OpenLiberty/open-liberty#17031]
The issue is uncovered after verifying https://github.com/OpenLiberty/open-liberty/issues/16072.
So far within wsscxf FAT Lite bucket, another 3 test methods failed the same:

testCxfCallerHttpPolicy_EE8_FEATURES    (com.ibm.ws.wssecurity.fat.cxf.caller.CxfCallerUNTTests)
testEcho3Service_EE8_FEATURES  (com.ibm.ws.wssecurity.fat.cxf.sample.CxfSymSampleTests)
testEcho6Service_EE8_FEATURES  ( com.ibm.ws.wssecurity.fat.cxf.sample.CxfSymSampleTests)

Below logs are extracted from the recreate of 
testWsComplexService_EE8_FEATURES  (com.ibm.ws.wssecurity.fat.cxf.x509migtoken.CxfX509MigTests)

log zip is in box folder: https://ibm.ent.box.com/file/806826631641 (requiring access permission to the box folder)

- messages.log
```
********************************************************************************
product = Open Liberty 21.0.0.5 (wlp-1.0.52.202104301748)
wlp.install.dir = C:/Users/Administrator/libertyGit/open-liberty/dev/build.image/wlp/
java.home = C:\jdk-11.0.11.9-openj9
java.version = 11.0.11
java.runtime = OpenJDK Runtime Environment (11.0.11+9)
os = Windows Server 2016 (10.0; amd64) (en_US)

...
[5/3/21, 12:33:13:723 PDT] 00000099 SystemOut                                                    O CxfX509MigSvcClient: wsdl:http://localhost:8010/x509migtoken/FatBAX21Service?wsdl
[5/3/21, 12:33:13:723 PDT] 00000099 SystemOut                                                    O thisMethod:testWsComplexService x509Policy:WsSignEncryptPolicy testMode:positive methodFull:testWsComplexService serviceName:{http://x509mig.libertyfat.test/contract}FatBAX21Service servicePort:{http://x509mig.libertyfat.test/contract}UrnX509Token21 servername:com.ibm.ws.wssecurity_fat.x509mig errorMsgVersion:null
...
[5/3/21, 12:33:25:288 PDT] 00000095 SystemOut                                                    O From CommonPasswordCallbackWss4j.java, using the callbacks:[Ljavax.security.auth.callback.Callback;@ddf671af
[5/3/21, 12:33:25:288 PDT] 00000095 SystemOut                                                    O From CommonPasswordCallbackWss4j.java: id:user1
[5/3/21, 12:33:25:288 PDT] 00000095 SystemOut                                                    O From CommonPasswordCallbackWss4j.java: Getting userPassword id:user1
[5/3/21, 12:33:25:489 PDT] 00000095 com.ibm.ws.logging.internal.impl.IncidentImpl                I FFDC1015I: An FFDC Incident has been created: "java.lang.NullPointerException org.apache.cxf.ws.security.wss4j.WSS4JInInterceptor 451" at ffdc_21.05.03_12.33.25.0.log
[5/3/21, 12:33:25:620 PDT] 00000095 SystemOut                                                    O Incoming Client Request as a SOAPMessage(mig bax21)
[5/3/21, 12:33:25:689 PDT] 00000095 SystemOut                                                    O From CommonPasswordCallbackWss4j.java, using the callbacks:[Lorg.apache.wss4j.common.ext.WSPasswordCallback;@6cbd52ec
[5/3/21, 12:33:25:689 PDT] 00000095 SystemOut                                                    O From CommonPasswordCallbackWss4j.java: id:x509ServerSecond
[5/3/21, 12:33:25:689 PDT] 00000095 SystemOut                                                    O From CommonPasswordCallbackWss4j.java: Getting signaturePassword id:x509ServerSecond
[5/3/21, 12:33:25:689 PDT] 00000095 SystemOut                                                    O From CommonPasswordCallbackWss4j.java: signaturePassword:KLibertyX509Server2

...
```

- ffdc
```
------Start of DE processing------ = [5/3/21, 12:33:25:357 PDT]
Exception = java.lang.NullPointerException
Source = org.apache.cxf.ws.security.wss4j.WSS4JInInterceptor
probeid = 451
Stack Dump = java.lang.NullPointerException
	at org.apache.cxf.ws.security.wss4j.WSS4JInInterceptor.importNewDomToSAAJ(WSS4JInInterceptor.java:424)
	at org.apache.cxf.ws.security.wss4j.WSS4JInInterceptor.handleMessageInternal(WSS4JInInterceptor.java:333)
	at org.apache.cxf.ws.security.wss4j.WSS4JInInterceptor.handleMessage(WSS4JInInterceptor.java:213)
	at org.apache.cxf.ws.security.wss4j.PolicyBasedWSS4JInInterceptor.handleMessage(PolicyBasedWSS4JInInterceptor.java:123)
	at org.apache.cxf.ws.security.wss4j.PolicyBasedWSS4JInInterceptor.handleMessage(PolicyBasedWSS4JInInterceptor.java:76)
	at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:308)
	at org.apache.cxf.transport.ChainInitiationObserver.onMessage(ChainInitiationObserver.java:123)
	at org.apache.cxf.transport.http.AbstractHTTPDestination.invoke(AbstractHTTPDestination.java:274)
	at com.ibm.ws.jaxws.endpoint.AbstractJaxWsWebEndpoint.invoke(AbstractJaxWsWebEndpoint.java:178)
	at com.ibm.ws.jaxws.webcontainer.LibertyJaxWsServlet.handleRequest(LibertyJaxWsServlet.java:134)
	at com.ibm.ws.jaxws.webcontainer.LibertyJaxWsServlet.doPost(LibertyJaxWsServlet.java:93)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:706)
	at com.ibm.ws.jaxws.webcontainer.LibertyJaxWsServlet.service(LibertyJaxWsServlet.java:85)
	at com.ibm.ws.webcontainer.servlet.ServletWrapper.service(ServletWrapper.java:1258)
	at com.ibm.ws.webcontainer.servlet.ServletWrapper.handleRequest(ServletWrapper.java:746)
	at com.ibm.ws.webcontainer.servlet.ServletWrapper.handleRequest(ServletWrapper.java:443)
	at com.ibm.ws.webcontainer.filter.WebAppFilterChain.invokeTarget(WebAppFilterChain.java:183)
	at com.ibm.ws.webcontainer.filter.WebAppFilterChain.doFilter(WebAppFilterChain.java:94)
	at com.ibm.ws.security.jaspi.JaspiServletFilter.doFilter(JaspiServletFilter.java:56)
	at com.ibm.ws.webcontainer.filter.FilterInstanceWrapper.doFilter(FilterInstanceWrapper.java:201)
	at com.ibm.ws.webcontainer.filter.WebAppFilterChain.doFilter(WebAppFilterChain.java:91)
	at com.ibm.ws.webcontainer.filter.WebAppFilterManager.doFilter(WebAppFilterManager.java:1002)
	at com.ibm.ws.webcontainer.filter.WebAppFilterManager.invokeFilters(WebAppFilterManager.java:1140)
	at com.ibm.ws.webcontainer.filter.WebAppFilterManager.invokeFilters(WebAppFilterManager.java:1011)
	at com.ibm.ws.webcontainer.servlet.CacheServletWrapper.handleRequest(CacheServletWrapper.java:75)
	at com.ibm.ws.webcontainer40.servlet.CacheServletWrapper40.handleRequest(CacheServletWrapper40.java:85)
	at com.ibm.ws.webcontainer.WebContainer.handleRequest(WebContainer.java:938)
	at com.ibm.ws.webcontainer.osgi.DynamicVirtualHost$2.run(DynamicVirtualHost.java:279)
	at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink$TaskWrapper.run(HttpDispatcherLink.java:1159)
	at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink.wrapHandlerAndExecute(HttpDispatcherLink.java:428)
	at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink.ready(HttpDispatcherLink.java:387)
	at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.handleDiscrimination(HttpInboundLink.java:566)
	at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.handleNewRequest(HttpInboundLink.java:500)
	at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.processRequest(HttpInboundLink.java:360)
	at com.ibm.ws.http.channel.internal.inbound.HttpICLReadCallback.complete(HttpICLReadCallback.java:70)
	at com.ibm.ws.tcpchannel.internal.WorkQueueManager.requestComplete(WorkQueueManager.java:504)
	at com.ibm.ws.tcpchannel.internal.WorkQueueManager.attemptIO(WorkQueueManager.java:574)
	at com.ibm.ws.tcpchannel.internal.WorkQueueManager.workerRun(WorkQueueManager.java:958)
	at com.ibm.ws.tcpchannel.internal.WorkQueueManager$Worker.run(WorkQueueManager.java:1047)
	at com.ibm.ws.threading.internal.ExecutorServiceImpl$RunnableWrapper.run(ExecutorServiceImpl.java:238)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:836)
```

**Comment by meiaus** [comment]
@arunavemulapalli

**Comment by arunavemulapalli** [comment]
Fix is included in #17452

**Comment by meiaus** [comment]
fixed; closing
